Source/WebCore/platform/graphics/cg/GradientRendererCG.cpp

```diff
@@ -485,11 +493,20 @@ GradientRendererCG::Strategy GradientRendererCG::makeGradient(ColorInterpolation
         // extended sRGB for all gradients.
         if (hasOnlyBoundedSRGBColorStops(stops)) {
             for (const auto& stop : stops) {
-                auto [r, g, b, a] = stop.color.toColorTypeLossy<SRGBA<float>>().resolved();
-                colorComponents.appendList({ r, g, b, a });
+                if (destinationColorSpace) {
```

Comment:
Why is this only happening in the all bounded sRGB case?

Reply:
Because that happens to be the case I'm targeting for optimization.